decidim-consultations/app/scrubbers/decidim/consultations/question_title_scrubber.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Consultations
+    class QuestionTitleScrubber < Decidim::UserInputScrubber
+      private
+
+      def custom_allowed_tags
+        Loofah::HTML5::SafeList::ACCEPTABLE_ELEMENTS - RESTRICTED_TAGS
```

Comment:
What is the difference of this compared to `UserInputScrubber`. Isn't this exactly the same?

But I think there should be less tags that we allow for the consultation questions.

The approach to solve this is completely fine, it's really clean!

Reply:
At the moment is the same with the `UserInputScrubber`, and i need to remove some more elements.